plugins/omelasticsearch/omelasticsearch.c

```diff
@@ -127,8 +133,9 @@ typedef struct wrkrInstanceData {
 /* tables for interfacing with the v6 config system */
 /* action (instance) parameters */
 static struct cnfparamdescr actpdescr[] = {
-	{ "server", eCmdHdlrGetWord, 0 },
+	{ "server", eCmdHdlrArray, 0 },
```

Comment:
yes, indeed it is. Also, if I understood correctly, the currently-existing tests should handle this, right?